lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -163,6 +165,35 @@ def add_invoice(post, money, options)
         post[:metadata][:udf5] = application_id || 'ActiveMerchant'
       end
 
+      def add_recipient_data(post, options)
+        return unless recipient = options[:recipient]
```

Comment:
We might want to add a check to make sure that options[:recipient] is a Hash -- otherwise the attempts to access data from within might throw NoMethodError if someone passes in a string